tests/components/wallbox/__init__.py

```diff
@@ -71,7 +72,7 @@
 )
 
 
-async def setup_integration(hass):
+async def setup_integration(hass: HomeAssistant):
```

Comment:
When typing a method, it needs to be fully typed (including return type).

Reply:
Just adding `-> None` should work. Could you also add return types in `test_lock.py`? Missed that in the earlier PR.

Reply:
Valid point, I now also included typing for the return value